aws/resource_aws_s3_bucket_object.go

```diff
@@ -194,7 +194,9 @@ func resourceAwsS3BucketObjectPut(d *schema.ResourceData, meta interface{}) erro
 		}
 		body = bytes.NewReader(contentRaw)
 	} else {
-		return fmt.Errorf("Must specify \"source\", \"content\", or \"content_base64\" field")
```

Comment:
I think the whole `else` block can be removed, leaving `body` as `nil`.

Reply:
👍